testing-commandments.md

```diff
@@ -0,0 +1,22 @@
+# HolidayExtras Automated Testing Commandments
+
+## Thou shalt be consistent within projects
+If a project has a de-facto standard, it should be maintained. The process for changing a de-facto standard should take the form of a whiteboard meeting, not a pull request.
```

Comment:
I think the "road map standard" should be the go to consideration where it exists, in a perfect world this will also be the de facto standard. For example on the trip planner we have a lot of unit tests, but the road map is to continue to use those where appropriate, but add more integration and system level tests.

"The process for changing a de-facto standard should take the form of a whiteboard meeting, not a pull request." I agree with the value of such a meeting, but I think a pull request, or pull requests are an important part of the process. 1. Because it gives visibility to those who were not in the white board session. 2. There might be details that come up in PRs that were missed or glossed over in any upfront meeting.

Potentially each project should have its own testing guidelines/info on current state/road map that expands on this one.